pkgs/tools/nix/options/options

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
```

Comment:
I am not sure about having the source code of this script inside `nixpkgs` source code. Reading PR https://github.com/NixOS/nixpkgs/pull/129696, it seems that `nixos-option` is inside `nixpkgs` because it is necessary to bootstrap the system.

This is not the case of this script, so I think it would be better to have it on a separate repository.